packages/expo-cli/src/commands/run/buildAndroidClientAsync.ts

```diff
@@ -0,0 +1,50 @@
+import { AndroidConfig } from '@expo/config-plugins';
+import spawnAsync from '@expo/spawn-async';
+import { Android } from '@expo/xdl';
+import ora from 'ora';
+import path from 'path';
+
+type Options = {
+  buildVariant: string;
+};
+
+function getGradleTask(buildVariant: string): string {
+  return `install${buildVariant.charAt(0).toUpperCase()}${buildVariant.slice(1)}`;
+}
```

Comment:
Add more types and make it explicitly clear what's going on here. Also iirc, people often use the `app:` prefix to target just the module? If there's reasoning to do/not do that, could you add it in a comment.